.gitignore

```diff
@@ -1,6 +1,7 @@
 ### IntelliJ IDEA ###
 out/
```

Comment:
Look, you've included this folder, but never used it
And your binary files after compilation now placed beside their .java files
You get mess
Please separate your source code (`src`) and your binary distribution files (`out`) during your compilation process

And after this, you won't need `*.class` filter here, only entire `out`
Eventually, your gitignore will look like this:
```
out/
*.iml
.idea/
.vscode/
```

Reply:
I said how your file should look like